### PR TITLE
Display start time on the worker summary screen. Fixes #419

### LINKED ISSUE #419: Display start time on the worker summary screen
@7a : At the moment, it is not easy to see what worker started when, when you use **OWTF** in a *large scope*, this is quite important because you don’t want a single tool to delay the scan. i.e. If I see that nikto is running for more than 1 day, something is wrong.

With this in mind: It would be nice if http://127.0.0.1:8009/ui/workers/ displayed something like `“Start: 2015/03/10 12:30 (~3 hours)”` to the right of the current “PID” field.

### PR DESCRIPTION
**Pull request by saganshul**
@delta24 @7a @DePierre Please review. Fixes #419 
PFA screenshot.

screenshot from 2016-03-12 20-27-48

### CODE REVIEW

File: framework/interface/templates/manager_interface.html [hunk 1]
```diff
@@ -128,6 +129,9 @@
                         html += '<div class="panel-heading"><h3 class="panel-title">Worker '+obj.id+getControlButtons(obj)+'</h3></div>';
                         html += '<div class="panel-body">';
                         html += '<p><strong>PID:</strong> '+obj.worker+'</p>';
+                        if ( obj.start_time.localeCompare("NA") == -1 ){
```
Comment (delta24) [inline framework/interface/templates/manager_interface.html]: Remove spacing between code and parenthesis.

**Comment by saganshul** [comment]
@delta24  PR updated. Please Review.

File: framework/interface/templates/manager_interface.html [hunk 2]
```diff
@@ -128,6 +129,12 @@
                         html += '<div class="panel-heading"><h3 class="panel-title">Worker '+obj.id+getControlButtons(obj)+'</h3></div>';
                         html += '<div class="panel-body">';
                         html += '<p><strong>PID:</strong> '+obj.worker+'</p>';
+                        if (obj.start_time.localeCompare("NA") == -1){
+                            html += '<p><strong>Start Time: </strong> '+obj.start_time+' (~ ' + getElapsedTime(obj.start_time)+')'+'</p>';
+                        }
+                        else{
```
Comment (delta24) [inline framework/interface/templates/manager_interface.html]: Use the if-else block like this:

```
if (condition) {
   statements1
} else {
   statements2
}
```

**Comment by delta24** [comment]
Looks good otherwise. Give me some time to manually lull and check first - until tonight :)

**Comment by delta24** [comment]
@saganshul Please use the user's local time to show the time it first started - you can do it very easily from javascript - right now I've to convert every time from UTC.

**Comment by saganshul** [comment]
@delta24, It is showing user's local time right now.
You can see in attached screenshot. My System time was 1:24 PM and start time of worker was also 13:24:50. 
Also start time is coming from python function strftime which gives me local time and also if you are talking about getElapsedTime function in manage_interface.html, it is giving me elapsed time i.e difference between start_time and present time hence it doesn't matter whether it is UTC or IST. 
Please correct me if I am understanding your comment wrongly. 

screenshot from 2016-03-14 13-24-51

**Comment by delta24** [comment]
Ah, ok- my bad. My Kali VM is set on a different timezone :/

**Comment by delta24** [comment]
Thank you @saganshul for the fix! :)

**Pull request merged**